boot/env.ml

```diff
@@ -58,7 +58,7 @@ let fail_lib lib =
   let open Printf in
   eprintf "File not found: %s\n" lib;
   eprintf "The path for Coq libraries is wrong.\n";
-  eprintf "Coq libraries are shipped in the coq-stdlib package.\n";
+  eprintf "Coq prelude is shipped in the rocq-init package.\n";
```

Comment:
```suggestion
  eprintf "The Rocq prelude is shipped in the rocq-init package.\n";
```

And Rocq on line 60, too.

Reply:
Renaming operations should be kept for separate PRs.